api/src/Entity/Period.php

```diff
@@ -245,4 +247,24 @@ public function removeMaterialItem(MaterialItem $materialItem): self {
 
         return $this;
     }
+
+    /**
+     * The day number of the first Day in period.
+     */
+    public function getFirstDayNumber(): int {
+        $expr = Criteria::expr();
+        $crit = Criteria::create();
+        $crit->where($expr->lt('start', $this->start));
```

Comment:
Erlauben wir aktuell überlappende Periods? Wenn ja, dann kriegen wir damit duplicates.

Reply:
Nein, erlauben wir explizit nicht

Reply:
Hmm, im alten Backend, ja (https://github.com/ecamp/ecamp3/pull/526). Aber ich glaub's nicht, dass wir das in der neuen API implementiert haben. Konnte auf jeden Fall keinen Validator dazu finden.

Reply:
Stimmt, aber ich glaube im Frontend verlassen wir uns am einen oder anderen Ort inzwischen auch darauf. Ich habs jetzt mal in #1579 als TODO erfasst.

Reply:
#2454 
Soll jeder Tag genau eine Nummer erhalten?
Oder jeder Datum?  Aber ScheduleEntry Nummer unique?
Wie soll im Fall von überlappenden Perioden nummeriert werden?